src/registry/mod.rs

```diff
@@ -247,15 +248,21 @@ where
     type Stream = GenerateBundles<P, B>;
     type LocalesIter = std::vec::IntoIter<LanguageIdentifier>;
 
-    fn bundles_iter(&self, locales: Self::LocalesIter, resource_ids: Vec<String>) -> Self::Iter {
+    fn bundles_iter(
+        &self,
+        locales: Self::LocalesIter,
+        resource_ids: Vec<ResourceId>,
+    ) -> Self::Iter {
+        let resource_ids = resource_ids.into_iter().map(ResourceId::from).collect();
```

Comment:
`nit`: You're accepting a `Vec<ResourceId>` and then mapping it onto a `Vec` of `ResourceId`?